resources/build/ci/trigger-build-bot.inc.sh

```diff
@@ -39,6 +39,34 @@ function test_bot_check_pr_body() {
   fi
 }
 
+#
+# Check the Keyman-Api-Check command whether or not to run API checks.
+# Any value other than 'skip' or no Keyman-Api-Check command will run
+# the API checks.
+#
+# Parameters:
+#   1: PR number (not currently used)
+#   2: PR JSON data from api.github.com/repos/keymanapp/keyman/pulls/#
+#
+function is_skip_keyman_api_check() {
+  local PRNUM=$1
+  local prinfo="$2"
+  local prbody prApiCheck
+
+  set -o noglob
+  IFS=$'\n'
+  prbody="$(echo "${prinfo}" | "${JQ}" -r '.body')"
+  prApiCheck="$(echo -e "${prbody}" | grep '^Keyman-Api-Check:' | cut -d: -f 2 - | tr -d '[:space:]')"
+  unset IFS
```

Comment:
prApiCheck="$(echo "${prinfo}" | "${JQ}" -r '.body["Keyman-Api-Check"]')"
```suggestion
  prbody="$(echo "${prinfo}" | "${JQ}" -r '.body')"
  prApiCheck="$(echo -e "${prbody}" | grep '^Keyman-Api-Check:' | cut -d: -f 2 - | tr -d '[:space:]')"
  unset IFS
```